deb-package-builder/extensions/grpc/grpc-fix-path.patch

```diff
@@ -0,0 +1,16 @@
+Index: gcp-php56-grpc-1.0.1-5.6.29/config.m4
+===================================================================
+--- gcp-php56-grpc-1.0.1-5.6.29.orig/config.m4
++++ gcp-php56-grpc-1.0.1-5.6.29/config.m4
+@@ -5,7 +5,7 @@ if test "$PHP_GRPC" != "no"; then
+   dnl Write more examples of tests here...
+ 
+   dnl # --with-grpc -> add include path
+-  PHP_ADD_INCLUDE(../../grpc/include)
+-  PHP_ADD_INCLUDE(../../grpc/src/php/ext/grpc)
+-  PHP_ADD_INCLUDE(../../grpc/third_party/boringssl/include)
++  PHP_ADD_INCLUDE(./include)
++  PHP_ADD_INCLUDE(./src/php/ext/grpc)
++  PHP_ADD_INCLUDE(./third_party/boringssl/include)
```

Comment:
Should this be a PR to `grpc/grpc`? It's strange because neither path exists, but the updated path at least exists when the file is copied to the root of the library, so it's a step up.